book.json

```diff
@@ -5,7 +5,8 @@
       "page-toc",
       "-sharing",
       "hints",
-      "versions"
+      "versions",
+      "alerts"
```

Comment:
We already had the `hints` plugin that did the job, but this one seems better. Changes needed therefore.